node/Cargo.toml

```diff
@@ -29,7 +29,6 @@ xcm-builder = { git = "https://github.com/purestake/polkadot", branch = "notlesh
 
 [features]
 default = [ "moonbase-native", "moonriver-native", "moonbeam-native" ]
-evm-tracing = [ "moonbeam-service/evm-tracing", "moonbeam-cli/evm-tracing" ]
```

Comment:
I think we should keep this feature here, even if it only makes sense for the runtime. It allows you to choose if you want tracing or not in the native runtime.

Reply:
I think the change makes sense, we only want that feature in the runtimes, and we don't care for Native that much, as it's something that parity wants to remove at some point.

Reply:
For the moment the native runtime is not removed by parity, and I don't see anything coming on that. It's still very useful for debugging, and we use it for the IC tests. So we have to continue to manage the native case for the moment and maybe forever.
You'll notice that we also have moonbase-native features, etc. at the client level. They don't change anything in the client code, but they change the native runtime code.

Reply:
Honestly I don't see the point of having that feature in the client now.

Reply:
This allows you to choose if you want tracing or not in the native runtime. It avoids to use a complex cargo command with several -p options for the IC or to test the tracing in the dev environment.
The client would remain unique in any case, I see the interest and I would like to make a small PR for that.